classes/image_types_ostree.bbclass

```diff
@@ -104,26 +104,30 @@ IMAGE_CMD_ostree () {
         ln -sf var/roothome root
     fi
 
-    checksum=`sha256sum ${DEPLOY_DIR_IMAGE}/${OSTREE_KERNEL} | cut -f 1 -d " "`
-
-    cp ${DEPLOY_DIR_IMAGE}/${OSTREE_KERNEL} boot/vmlinuz-${checksum}
+    FILES_TO_CHECKSUM="${DEPLOY_DIR_IMAGE}/${OSTREE_KERNEL} ${INITRAMFS_IMAGE}-${MACHINE}.${INITRAMFS_FSTYPES} ${KAKUS}"
```

Comment:
Sorry, what is `KAKUS`? I can't find it defined anywhere.

Reply:
sorry about this one. Its leftover from testing.